README.md

```diff
@@ -38,25 +38,24 @@ pip install git+https://github.com/aristanetworks/j2lint.git
 
 To get started with j2lint code, clone the Jinja2 Linter project on your system:
 
-```
-git clone https://gitlab.aristanetworks.com/ansible-team/jinja2-linter
+```bash
+git clone https://github.com/aristanetworks/j2lint.git
 ```
 
 ### Prerequisites
 
-1. Python version 3.6+
-
+0\. Python version 3.6+
```

Comment:
Please go through the numbering changes again. When I preview the file, the numbers are not correct.